pkg/workflow/spec/report.go

```diff
@@ -0,0 +1,4 @@
+package spec
+
+type SpecValidationErrorReporter struct {
```

Comment:
oh yeah, I'll remove this for now. originally it was meant for getting the validation errors back to the controller so it could put them in the resource status field. I'll implement this later after we catch issues in sentry for a bit.